nominal/cli/dataset.py

```diff
@@ -64,3 +67,45 @@ def get(rid: str, base_url: str, token: str | None) -> None:
     client = get_client(base_url, token)
     dataset = client.get_dataset(rid)
     print(dataset)
+
+
+@dataset_cmd.command("summarize")
+@click.option("-r", "--rid", required=True, multiple=True, help="RID(s) of the dataset(s) to summarize")
+@click.option(
+    "-c",
+    "--csv",
+    type=click.Path(exists=False, dir_okay=False, resolve_path=True, path_type=pathlib.Path),
```

Comment:
wdyt about having `nom dataset summarize` having an output format choice (table, csv) and always printing to stdout? then users can pipe that into a file if they want to write to a file, and it can be extended to include e.g. json or another machine readable format in the future if necessary.

Reply:
the `-o` vs stdout is just a taste thing. personally I just lean towards fewer branches / less special-casing, so instead of "if no `-o` flag, print to stdout" or "use `-o -` for stdout" I try to have uniform behavior in all cases.